### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54215] [Code Security]
### Select Topic Area

Product Feedback

### Body

The move to mandatory 2FA is a blatant attempt by Microsoft to harvest more personal information. Their claims that it's to "Secure the supply chain" are proven obviously false by the fact that they're not touching anything at all related to personal access tokens, which bypass 2FA entirely and are just as bad, if not potentially worse, than a username and password. 

In short, unless this decision is reverted, I encourage every person reading this to move to a git provider that's not associated with a company well known for harvesting and selling its users' data.
